core/embed/trezorhal/stm32u5/secret.c

```diff
@@ -143,6 +143,17 @@ secbool secret_optiga_present(void) {
   return secret_present(SECRET_OPTIGA_KEY_OFFSET, SECRET_OPTIGA_KEY_LEN);
 }
 
+secbool secret_optiga_set(const uint8_t *secret) {
```

Comment:
When passing fixed-size arrays, let's use array declarations rather than pointers in all new code, e.g. `secret_optiga_set(const uint8_t secret[SECRET_OPTIGA_KEY_LEN])`, here and in the header files as well. Same applies to `secret_optiga_get()`. This makes no difference in terms of the resulting compiled code. It is mainly a form of documentation, saving time of devs trying to figure out the expected array size. However, some compilers use it to issue warnings when a smaller array is supplied.

Reply:
fixed https://github.com/trezor/trezor-firmware/commit/3bc37ecfbed67857165a82abd8e20f022ca05b12